### PR TITLE
Unified composer updates

### PR DESCRIPTION
**Pull request by Reettaphant**
Use new endpoint in composer to perform updates instead of making separate requests to separate composer endpoints. 

see https://github.com/guardian/flexible-content/pull/2978

### CODE REVIEW

File: public/video-ui/src/util/getComposerData.js [hunk 1]
```diff
@@ -6,102 +6,27 @@ function asBooleanString(value) {
...
+    trailText: video.trailText ? video.trailText : null,
+    linkText: video.title,
+    sensitive: asBooleanString(video.sensitive),
+    legallySensitive: asBooleanString(video.legallySensitive),
+    blockAds: asBooleanString(video.blockAds),
+    commentable: asBooleanString(video.composerCommentsEnabled),
+    relatedContentOff: asBooleanString(video.suppressRelatedContent),
+    seoOptimised: asBooleanString(video.optimisedForWeb),
+    commissioningDesks: video.commissioningDesks.join('|'),
+    byline: video.byline.join('|'),
```
Comment (akash1810) [inline public/video-ui/src/util/getComposerData.js]: as mentioned in the Composer PR, could we send this as an array?

File: public/video-ui/src/util/getComposerData.js [hunk 2]
```diff
@@ -6,102 +6,27 @@ function asBooleanString(value) {
...
-    value: asBooleanString(true),
-    belongsTo: 'settings'
+  return {
+    headline: video.title,
+    standfirst: video.description ? video.description : null,
+    trailText: video.trailText ? video.trailText : null,
+    linkText: video.title,
+    sensitive: asBooleanString(video.sensitive),
+    legallySensitive: asBooleanString(video.legallySensitive),
+    blockAds: asBooleanString(video.blockAds),
```
Comment (akash1810) [inline public/video-ui/src/util/getComposerData.js]: as mentioned in the Composer PR, could we send this as a boolean?

File: public/video-ui/src/util/getComposerData.js [hunk 3]
```diff
@@ -1,107 +1,26 @@
...
+    relatedContentOff: video.suppressRelatedContent,
+    seoOptimised: video.optimisedForWeb,
+    commissioningDesks: video.commissioningDesks.join('|'),
+    byline: video.byline.join('|'),
+    keywords: video.keywords.join('|'),
+    thumbnail: video.trailImage && video.trailImage.assets.length > 0
+      ? parseComposerDataFromImage(video.trailImage, video.trailText)
+      : null,
+    expiryDate: video.expiryDate,
+    embargoedIndefinitely: isTrainingMode ? true : false
```
Comment (akash1810) [inline public/video-ui/src/util/getComposerData.js]: don't need the ternary anymore

Comment (Reettaphant) [reply public/video-ui/src/util/getComposerData.js]: why not?

Comment (akash1810) [reply public/video-ui/src/util/getComposerData.js]: we can do `embargoedIndefinitely: isTrainingMode` instead

**Pull request merged**